lib/constructs/persistence/data-pipeline.ts

```diff
@@ -202,4 +202,54 @@ export class SyllabusSyncPipeline extends AbstractDataPipeline {
             filters: [{prefix: 'syllabus/'}],
         }))
     }
-}
\ No newline at end of file
+}
+
+// update blog on notification
+export class BlogContentPipeline extends AbstractDataPipeline {
+
+    readonly dataSource: Bucket;
+
+    readonly processor: Function;
+
+    readonly dataWarehouse: Table;
+
+    constructor(scope: cdk.Construct, id: string, props?: DataPipelineProps) {
+        super(scope, id);
+
+        this.dataWarehouse = new Table(this, 'blog-table', {
+            partitionKey: {name: "title", type: AttributeType.STRING},
+            sortKey: {name: "ctime", type: AttributeType.STRING},
+            billingMode: BillingMode.PROVISIONED,
+            encryption: TableEncryption.DEFAULT,
+            removalPolicy: cdk.RemovalPolicy.RETAIN,
+            timeToLiveAttribute: "ttl",
```

Comment:
We do not need a ttl attribute for record invalidation.

Reply:
Delete timeToLiveAttribute